dash-pipeline/SAI/specs/dash_routing_group.yaml

```diff
@@ -0,0 +1,24 @@
+!!python/object:utils.sai_spec.sai_api_group.SaiApiGroup
+name: dash_routing_group
```

Comment:
this looks to be unexpected, is this something we forgot to clear?

Reply:
Removed the file since this is part of OB routing.